ci.sh

```diff
@@ -16,7 +16,10 @@ run() {
 
 mkdir -p "${TMPDIR}"
 
-# build nixpkgs-fmt
+# build nixpkgs-fmt with nix
+run nix-build .
+
+# build nixpkgs-fmt with cargo
 run cargo build --verbose
```

Comment:
`nix-build .` compiles `nixpkgs-fmt` in release mode. `cargo build` compiles it again in debug mode.

I wanted to be careful and conservative here, but running `cargo build` seems unnecessary given that `nix-build .` already compiles the whole project. @Mic92 @zimbatm should I leave `cargo build` in here or remove it, what do you think?

Reply:
If I remember correctly, the pre-commit hook below depends on having nixpkgs-fmt being compiled as it runs it against our own source code.

Reply:
It's using `cargo run --` so it will build nixpkgs-fmt in any case:
https://github.com/nix-community/nixpkgs-fmt/blob/5af63196a40632dafa43311f12e8ab6aa78d28b7/.pre-commit-config.yaml#L12-L24

Reply:
So it could be dropped as well.

Reply:
My understanding of your comments is that you agree that the separate `cargo build` step is not necessary. I've removed it.